Source/Engine/Syntax/SearchTargetSyntax.cs

```diff
@@ -11,26 +11,19 @@
 
 namespace Nezaboodka.Nevod
 {
-    public class SearchTargetSyntax : Syntax
+    public abstract class SearchTargetSyntax : Syntax
     {
         public new string SearchTarget { get; }
```

Comment:
I know it was here before, but can you please remove `new`, looks like it's redundant.
```suggestion
        public string SearchTarget { get; }
```